lms/static/sass/views/_account-settings.scss

```diff
@@ -36,45 +39,269 @@
 .wrapper-account-settings {
 
     .wrapper-header {
+        max-width: grid-width(12);
+        height: 139px;
+        border-bottom: 4px solid #f5f5f5;
 
         .header-title {
             @extend %t-title4;
             margin-bottom: ($baseline/2);
+            padding-top: ($baseline*2);
         }
 
         .header-subtitle {
             color: $gray-l2;
         }
+
+        .account-nav {
+        margin: ($baseline/2) 0px;
+        padding: 0px;
+        list-style: none;
+        @include float(left);
+        .account-nav-item{
+            display: flex;
+            margin: 0px;
+            text-transform: none;
+            @include float(left);
+            justify-content: center;
+            .account-nav-link{
+                font-size: em(16);
+                color: #7a7979;
+                padding: 5px 25px 23px;
+                display: inline-block;
+            }
+            .active{
+                border-bottom: 4px solid $courseware-border-bottom-color !important;
+            }
+        }
+        li,
+        div {
+            display: inline-block;
+            margin: 0 $baseline+1 0 0;
+            font-size: em(14);
+            font-weight: 600;
```

Comment:
Please use the already defined font-weight variables https://github.com/edx/edx-platform/blob/76b8e2e89761c30ac7aa2fa5e8aca4dc35d9002b/lms/static/sass/base/_variables.scss#L320